resources/js/views/Statistics.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+	<Toolbar class="w-full border-0 h-14">
+		<template #start>
+			<router-link :to="{ name: 'gallery' }">
+				<Button icon="pi pi-angle-left" class="mr-2" severity="secondary" text />
+			</router-link>
+		</template>
+
+		<template #center>
+			{{ "Statistics" }}
+		</template>
+
+		<template #end> </template>
+	</Toolbar>
+	<Panel v-if="is_se_preview_enabled" class="text-center">
+		This is a preview of the upcoming statistics page.<br />
```

Comment:
I think this is no longer "upcoming"? You may have dealt with this in the stack that's waiting for me... I guess the "Lychee SE" could be a link too...
```suggestion
		This is a preview of the statistics page available in Lychee SE.<br />
```

Reply:
Nah I think we don't need a link for SE on that one. :)
We already have enough of those.